.storybook/preview.js

```diff
@@ -1,4 +1,6 @@
 import '../css/dialtone-globals.less';
+import '@dialpad/dialtone/lib/dist/css/dialtone.min.css';
+import '@dialpad/dialtone-tokens/dist/css/variables-light.css';
```

Comment:
hmm I think we should export variables-light and variables-dark as part of the dialtone.min.css. They would still be isolated within their classes.

Reply:
Sounds good, I'll add this to the dark mode PR on dialtone